src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -99,35 +122,59 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
               ></paper-progress>
             `}
         <div class="controls">
-          <paper-icon-button
-            icon="hass:power"
-            .action=${stateObj.state === "off" ? "turn_on" : "turn_off"}
-            @click=${this._handleClick}
-          ></paper-icon-button>
+          ${(stateObj.state === "off" &&
+            !supportsFeature(stateObj, SUPPORT_TURN_ON)) ||
+          (stateObj.state === "on" &&
+            !supportsFeature(stateObj, SUPPORT_TURN_OFF))
+            ? ""
+            : html`
+                <paper-icon-button
+                  .icon=${"hass:power"}
+                  .action=${stateObj.state === "off" ? "turn_on" : "turn_off"}
+                  @click=${this._handleClick}
+                ></paper-icon-button>
+              `}
           <div class="playback-controls">
-            <paper-icon-button
-              icon="hass:skip-previous"
-              .action=${"media_previous_track"}
-              @click=${this._handleClick}
-            ></paper-icon-button>
-            <paper-icon-button
-              class="playPauseButton"
-              icon=${stateObj.state !== "playing"
-                ? "hass:play"
-                : supportsFeature(stateObj, SUPPORT_PAUSE)
-                ? "hass:pause"
-                : "hass:stop"}
-              .action=${"media_play_pause"}
-              @click=${this._handleClick}
-            ></paper-icon-button>
-            <paper-icon-button
-              icon="hass:skip-next"
-              .action=${"media_next_track"}
-              @click=${this._handleClick}
-            ></paper-icon-button>
+            ${!supportsFeature(stateObj, SUPPORT_PREVIOUS_TRACK)
+              ? ""
+              : html`
+                  <paper-icon-button
+                    .icon=${"hass:skip-previous"}
+                    .disabled="${OFF_STATES.includes(stateObj.state)}"
+                    .action=${"media_previous_track"}
+                    @click=${this._handleClick}
+                  ></paper-icon-button>
+                `}
+            ${(stateObj.state !== "playing" &&
+              !supportsFeature(stateObj, SUPPORTS_PLAY)) ||
+            stateObj.state === "unavailable"
```

Comment:
Use the const for `unavailable` -> https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/data/entity.ts#L1